docs/releases/6.0.txt

```diff
@@ -37,6 +37,48 @@ compatible with Django 6.0.
 What's new in Django 6.0
 ========================
 
+Content Security Policy (CSP) Support
+-------------------------------------
+
+Django 6.0 introduces built-in :ref:`Content Security Policy (CSP)
+<security-csp>` support, making it easier to protect your application against
+various types of content injection attacks. CSP helps you specify which content
+sources are allowed to be loaded by the browser.
```

Comment:
```suggestion
The new built-in :ref:`Content Security Policy (CSP) <security-csp>` support
makes it easier to protect your application against content injection attacks.
CSP is a security standard that allows you to specify which resources browsers
are allowed to load.
```

We don't normally say "Django <VERSION>" throughout the release notes. Also, I edited for brevity clarity.